src/test/java/org/apache/datasketches/tuple/strings/ArrayOfStringsSketchTest.java

```diff
@@ -46,11 +46,11 @@ public void checkSketch() {
       sketch1.update(strArrArr[i], strArrArr[i]);
     }
     sketch1.update(strArrArr[0], strArrArr[0]); //insert duplicate
-    //printSummaries(sketch1.iterator());
+    printSummaries(sketch1.iterator());
```

Comment:
did you mean to add this back in? (yes is fine, but if no just wanna flag the)

Reply:
Yes. Printing is disabled at the bottom of the class.